cmd/cosign/cli/upload/blob.go

```diff
@@ -102,6 +102,10 @@ func BlobCmd(ctx context.Context, files []cremote.File, contentType, imageRef st
 	if err != nil {
 		return err
 	}
+	// TODO: error format?
+	if dgster == nil {
+		return fmt.Errorf("dgstr is nil, no files uploaded?")
```

Comment:
I think you can use "errors.New" here instead of fmt.Errorf since there is no format parameter.